src/query/functions/src/aggregates/adaptors/aggregate_ornull_adaptor.rs

```diff
@@ -173,16 +173,16 @@ impl AggregateFunction for AggregateFunctionOrNullAdaptor {
     }
 
     #[inline]
-    fn deserialize(&self, place: StateAddr, reader: &mut &[u8]) -> Result<()> {
-        let flag = reader[reader.len() - 1];
```

Comment:
The old logic is wrong, is there any case can produce the wrong result?

Reply:
the old logic is correct, the new `merge` is  `old deserialize` + `old merge`